android/src/main/java/com/tailscale/ipn/App.kt

```diff
@@ -146,6 +146,10 @@ class App : Application(), libtailscale.AppContext {
         .editPrefs(Ipn.MaskedPrefs().apply { WantRunning = wantRunning }, callback)
   }
 
+  override fun setStopRunning(){
```

Comment:
this isn't formatted the same way as line 155

Does Kotlin not have an auto-formatter?